src/NServiceBus.Transport.SQS/Configure/S3Settings.cs

```diff
@@ -87,6 +87,13 @@ public S3Settings(string bucketForLargeMessages, string keyPrefix, IAmazonS3 s3C
         /// </summary>
         public string KeyPrefix { get; }
 
+        /// <summary>
+        /// When set to <c>true</c> the <c>PutObjectRequest</c>to store the message
+        /// body is not signed. This is useful to support services such as Cloudflare R2
+        /// that don't support payload signing.
+        /// </summary>
+        public bool? DisablePayloadSigning { get; set; }
```

Comment:
Do you know why you choose a nullable (tristate)?

Reply:
I mirrored the S3 SDK property type, no reasons at all. It seems, according to the documentation, that null means use whatever default for the SDK

Reply:
Ah OK. Make sense then 👍